_posts/en/newsletters/2025-11-21-newsletter.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "Bitcoin Optech Newsletter #381"
+permalink: /en/newsletters/2025/11/21/
+name: 2025-11-21-newsletter
+slug: 2025-11-21-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+
+FIXME:bitschmidty
+
+## News
+
+- **Modeling stale rates by propagation delay and mining centralization:**
+  Antoine Poinsot [posted to
+  delving][antoine delving] about modeling stale block rates and how
+  block propagation time affects a miner's revenue as a function of its
+  hashrate. He set up a base case scenario where all miners act
+  rationally, meaning that if they received a new block, then they
+  would immediately start mining on top of it and publish immediately.
+  This would lead to a revenue that is proportional to their share of
+  the hashrate.
+
+  In their model with uniform block propagation, they had two
+  situations.
+  1. Another miner found a block **before** we did. All other miners
+     received the competing miner's block first and started mining on
+     top of it. Any of these miners can then find a second block on
+     top.
+
+  2. Another miner finds a block **after** we did . It immediately
+     starts mining on top of it. Then the following block is also
+     found by this same miner.
+
+  Antoine points out that it's clear that between these situations, it is
+  more likely for a block to become stale in situation 1. And suggests
+  that miners care more about hearing others' blocks faster
+  than they care about publishing their own. He also suggests that the
+  probability of situation 2 increases significantly with miner
+  centralization. While it is useful to the probability decreases as miner
+  hashrate goes up for both situations, but Antoine wanted to compute
+  by how much.
+
+  To do this. he created the following two models.
+
+  Where **h** is the share of network hashrate, **s** is the number of seconds the
+  rest of the network found the same block before it did, **H** is a set of
+  hashrate on the network representing its distribution.
+
+  Model for situation 1:
+  <img src="https://latex.codecogs.com/svg.latex?\large\bold%20P(Before)=(1-\exp(-\dfrac{(1-h)\cdot s}{600}))\cdot(1-h)" alt="P(Before)">
+
+  Model for situation 2:
+  <img src="https://latex.codecogs.com/svg.latex?\large\bold%20P(After_{i})=\sum_{1\leq j\leq|H|}^{j\neq i}(1-\exp(-\dfrac{h_{j}\cdot t}{600}))\cdot h_j" alt="P(After_i)">
+
+  In the delving post, he went on to show graphs of hashrate similar to
+  that of the past 3 months and their corresponding impact of
+  propagation time on mining centralization. The graphs show how
+  centralized miners gain significantly more the longer the propagation
```

Comment:
```suggestion
  propagation time on mining centralization. The graphs show how
  larger miners gain significantly more the longer the propagation
```

In this case "centralized" is not really an attribute, all mining pools are technically centralized. It's a force: mining operations are attracted toward larger mining pools, making them even larger and more attractive, etc..